### ISSUE TITLE
Suggestion: Option to auto-select first emoji result

### ISSUE THREAD

**Issue by ConnorJC3** [ajbura/cinny#1039]
### Describe the problem

When typing emoji, my muscle memory, built up over years of using Slack, is to type `:`, followed by the start of the emoji name, followed by Enter. On Cinny, this sends the message halfway through using it. Even after using Cinny for some time, I still accidentally send a message with emojis in it probably close to half the time.

### Describe the solution you'd like

Make the emoji selection work like slack: the first option is automatically selected after you start typing an emoji, and you can press enter to finish the autocomplete. Of course, not everyone will like this, so probably a toggle between this and the old behavior is appropriate.

### Alternatives considered

This could _maybe_ be done with a user script, but it would be janky and prone to breakage.

### Additional context

Willing to PR this if it would be accepted.

**Comment by LinuxinaBit** [comment]
Tab is what is currently used, and I agree that should probably be changed to that kind of autocomplete.
An extra bonus feature would be arrow key navigation so you don’t even have to touch your mouse!

**Comment by ajbura** [comment]
Does it still a issue as you can select emoji suggestion with `Tab` ?

**Comment by LinuxinaBit** [comment]
Not really, looks good 👍 
I'll just have to get used to using the tab key now...

**Comment by kfiven** [comment]
Closing as this resolved with selecting emojis/rooms/userids with tab.
